Lib/queue.py

```diff
@@ -67,8 +78,13 @@ def task_done(self):
 
         Raises a ValueError if called more times than there were items
         placed in the queue.
+
+        Raises a ShutDown if shutdown_state attribute is set to
+        `_queue_shutdown_immediate`.
```

Comment:
```suggestion
        Raises ShutDown if the queue has been shut down immediately.
```

Doc-strings are for humans